EIPS/eip-6785.md

```diff
@@ -0,0 +1,205 @@
+---
+eip: 6785
+title: ERC-721 Utilities Information Extension
+description: Provide easy access to information about the `utility` of an NFT via functions and the NFT's metadata
+author: Otniel Nicola (@OT-kthd), Bogdan Popa (@BogdanKTHD)
+discussions-to: https://ethereum-magicians.org/t/eip-6785-erc-721-utilities-extension/13568
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-03-27
+requires: 165, 721
+---
+
+## Abstract
+
+This specification defines standard functions and an extension of the metadata schema that outline what a 
+token's utility entails and how the utility may be used and/or accessed.
+This specification is an optional extension of [ERC-721](./eip-721.md).
+
+## Motivation
+
+This specification aims to bring clarity what the utility associated with an NFT is and how to access this utility.
+Relying on third-party platforms to obtain information regarding the utility of the NFT that one owns can lead to scams,
+phishing or other forms of fraud.
+
+Currently, utilities that are offered with NFTs, are not captured on-chain. We want the utility of an NFT to be part of
+the metadata of an NFT, and thus immutable. The metadata information would include: a) type of utility, b) description
```

Comment:
I'd be careful about claiming immutability here, since you're using URIs. The content located by the URI can be changed, unless it's a `data:`, `ipfs:`, or similarly immutable scheme.